### PR TITLE
Remove unneccessary imports

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Prepare for https://github.com/ansible-collections/news-for-maintainers/issues/34.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

File: plugins/plugin_utils/action_module.py [hunk 1]
```diff
@@ -69,9 +69,9 @@
     # For ansible-core 2.11, we can use the ArgumentSpecValidator. We also import
     # ModuleArgumentSpecValidator since that indicates that the 'classical' approach
     # will no longer work.
-    from ansible.module_utils.common.arg_spec import (
+    from ansible.module_utils.common.arg_spec import (  # noqa: F401, pylint: disable=unused-import
```
Comment (kristianheljas) [inline plugins/plugin_utils/action_module.py:72]: Whatever you decide in [community.crypto#569](https://github.com/ansible-collections/community.crypto/pull/569#discussion_r1098527315), apply the same here.

**Review by kristianheljas — APPROVED** [review]
Trusting your decision about the [import warning exclusion](https://github.com/ansible-collections/community.sops/pull/134/files#r1101140209), LGTM!

File: plugins/plugin_utils/action_module.py (hunk 1 above)
Comment (felixfontein) [reply plugins/plugin_utils/action_module.py:72]: I've added a comment in e65b55327d7b34b7b93e7b32731b0560989e9165.

**Review by kristianheljas — APPROVED** [review]
Thanks @felixfontein for making it clear what needs to be ignored, LGTM!

File: plugins/plugin_utils/action_module.py (hunk 1 above)
Comment (kristianheljas) [reply plugins/plugin_utils/action_module.py:72]: /resolved

**Comment by felixfontein** [comment]
@kristianheljas thanks for reviewing also this one!

**Pull request merged**